<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rabbit Remote Control</string>
  </property>
  <property name="windowIcon">
   <iconset theme="app"/>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionUpdate"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View(&amp;V)</string>
    </property>
    <widget class="QMenu" name="menuType">
     <property name="title">
      <string>Type</string>
     </property>
     <property name="icon">
      <iconset theme="view-type"/>
     </property>
     <addaction name="actionViewTab"/>
     <addaction name="actionViewSplit"/>
    </widget>
    <widget class="QMenu" name="menuActivity">
     <property name="toolTip">
      <string>Activities</string>
     </property>
     <property name="statusTip">
      <string>Activities</string>
     </property>
     <property name="title">
      <string>Activities</string>
     </property>
     <property name="icon">
      <iconset theme="network-wired"/>
     </property>
    </widget>
    <addaction name="actionFull_screen_F"/>
    <addaction name="menuType"/>
    <addaction name="actionTabBar_B"/>
    <addaction name="actionToolBar_T"/>
    <addaction name="actionStatus_bar_S"/>
    <addaction name="actionMain_menu_bar_M"/>
    <addaction name="separator"/>
    <addaction name="menuActivity"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuOperate">
    <property name="title">
     <string>Operate(&amp;O)</string>
    </property>
    <widget class="QMenu" name="menuStart">
     <property name="title">
      <string>Start(&amp;S)</string>
     </property>
     <property name="icon">
      <iconset theme="media-playback-start"/>
     </property>
    </widget>
    <addaction name="menuStart"/>
    <addaction name="actionStop"/>
    <addaction name="actionOpenListRecent"/>
    <addaction name="actionOpenRRCFile"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_to_favorite"/>
    <addaction name="actionClone"/>
    <addaction name="separator"/>
    <addaction name="actionExit_E"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools(&amp;T)</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuOperate"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenListRecent"/>
   <addaction name="separator"/>
   <addaction name="actionAdd_to_favorite"/>
   <addaction name="actionClone"/>
   <addaction name="separator"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionFull_screen_F"/>
   <addaction name="actionTabBar_B"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
   <addaction name="separator"/>
   <addaction name="actionExit_E"/>
  </widget>
  <action name="actionAbout">
   <property name="icon">
    <iconset theme="help-about"/>
   </property>
   <property name="text">
    <string>About(&amp;A)</string>
   </property>
   <property name="iconText">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
   <property name="statusTip">
    <string>About</string>
   </property>
   <property name="whatsThis">
    <string>About</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="icon">
    <iconset theme="system-software-update"/>
   </property>
   <property name="text">
    <string>Update(&amp;U)</string>
   </property>
   <property name="iconText">
    <string>Update</string>
   </property>
   <property name="toolTip">
    <string>Update</string>
   </property>
   <property name="statusTip">
    <string>Update</string>
   </property>
   <property name="whatsThis">
    <string>Update</string>
   </property>
  </action>
  <action name="actionToolBar_T">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ToolBar(&amp;T)</string>
   </property>
   <property name="iconText">
    <string>ToolBar</string>
   </property>
   <property name="toolTip">
    <string>ToolBar</string>
   </property>
   <property name="statusTip">
    <string>ToolBar</string>
   </property>
   <property name="whatsThis">
    <string>ToolBar</string>
   </property>
  </action>
  <action name="actionFull_screen_F">
   <property name="icon">
    <iconset theme="view-fullscreen"/>
   </property>
   <property name="text">
    <string>Full screen(&amp;F)</string>
   </property>
   <property name="iconText">
    <string>Full screen</string>
   </property>
   <property name="toolTip">
    <string>Full screen</string>
   </property>
   <property name="statusTip">
    <string>Full screen</string>
   </property>
   <property name="whatsThis">
    <string>Full screen</string>
   </property>
  </action>
  <action name="actionExit_E">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="window-close"/>
   </property>
   <property name="text">
    <string>Exit(&amp;E)</string>
   </property>
   <property name="iconText">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
   <property name="statusTip">
    <string>Exit</string>
   </property>
   <property name="whatsThis">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset theme="media-playback-stop"/>
   </property>
   <property name="text">
    <string>Stop(&amp;P)</string>
   </property>
   <property name="iconText">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop</string>
   </property>
   <property name="statusTip">
    <string>Stop</string>
   </property>
   <property name="whatsThis">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionOpenRRCFile">
   <property name="icon">
    <iconset theme="document-open"/>
   </property>
   <property name="text">
    <string>Open(&amp;O) rabbit remote control file</string>
   </property>
   <property name="iconText">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open rabbit remote control file</string>
   </property>
   <property name="statusTip">
    <string>Open rabbit remote control file</string>
   </property>
   <property name="whatsThis">
    <string>Open rabbit remote control file</string>
   </property>
  </action>
  <action name="actionTabBar_B">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="tabbar"/>
   </property>
   <property name="text">
    <string>TabBar(&amp;B)</string>
   </property>
   <property name="iconText">
    <string>TabBar</string>
   </property>
   <property name="toolTip">
    <string>TabBar</string>
   </property>
   <property name="statusTip">
    <string>TabBar</string>
   </property>
   <property name="whatsThis">
    <string>TabBar</string>
   </property>
  </action>
  <action name="actionZoom_window_to_remote_desktop">
   <property name="icon">
    <iconset theme="folder-remote"/>
   </property>
   <property name="text">
    <string>Zoom window to remote desktop</string>
   </property>
   <property name="statusTip">
    <string>Zoom window to remote desktop</string>
   </property>
   <property name="whatsThis">
    <string>Zoom window to remote desktop</string>
   </property>
  </action>
  <action name="actionScreenshot">
   <property name="icon">
    <iconset theme="camera-photo"/>
   </property>
   <property name="text">
    <string>Screenshot</string>
   </property>
   <property name="toolTip">
    <string>Screenshot</string>
   </property>
   <property name="statusTip">
    <string>Screenshot</string>
   </property>
   <property name="whatsThis">
    <string>Screenshot</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset theme="emblem-system"/>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="statusTip">
    <string>Settings</string>
   </property>
   <property name="whatsThis">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionClone">
   <property name="icon">
    <iconset theme="edit-copy"/>
   </property>
   <property name="text">
    <string>Clone</string>
   </property>
   <property name="toolTip">
    <string>Clone</string>
   </property>
   <property name="statusTip">
    <string>Clone</string>
   </property>
   <property name="whatsThis">
    <string>Clone</string>
   </property>
  </action>
  <action name="actionAdd_to_favorite">
   <property name="icon">
    <iconset theme="emblem-favorite"/>
   </property>
   <property name="text">
    <string>Add to favorites</string>
   </property>
   <property name="toolTip">
    <string>Add to favorites</string>
   </property>
   <property name="statusTip">
    <string>Add to favorites</string>
   </property>
   <property name="whatsThis">
    <string>Add to favorites</string>
   </property>
  </action>
  <action name="actionOpenListRecent">
   <property name="icon">
    <iconset theme="folder-open"/>
   </property>
   <property name="text">
    <string>Open the recent list(&amp;R)</string>
   </property>
   <property name="iconText">
    <string>Open the recent list</string>
   </property>
   <property name="toolTip">
    <string>Open the recent list</string>
   </property>
   <property name="statusTip">
    <string>Open the recent list</string>
   </property>
   <property name="whatsThis">
    <string>Open the recent list</string>
   </property>
  </action>
  <action name="actionMain_menu_bar_M">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="menu"/>
   </property>
   <property name="text">
    <string>Main menu bar(&amp;M)</string>
   </property>
   <property name="iconText">
    <string>Main menu bar</string>
   </property>
   <property name="toolTip">
    <string>Main menu bar</string>
   </property>
   <property name="statusTip">
    <string>Main menu bar</string>
   </property>
   <property name="whatsThis">
    <string>Main menu bar</string>
   </property>
  </action>
  <action name="actionStatus_bar_S">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Status bar(&amp;S)</string>
   </property>
   <property name="iconText">
    <string>Status bar</string>
   </property>
   <property name="toolTip">
    <string>Status bar</string>
   </property>
   <property name="statusTip">
    <string>Status bar</string>
   </property>
   <property name="whatsThis">
    <string>Status bar</string>
   </property>
  </action>
  <action name="actionRecordVideo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="media-record"/>
   </property>
   <property name="text">
    <string>Record video</string>
   </property>
  </action>
  <action name="actionViewTab">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Tab</string>
   </property>
  </action>
  <action name="actionViewSplit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Split</string>
   </property>
   <property name="toolTip">
    <string>Split view</string>
   </property>
   <property name="statusTip">
    <string>Split view</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
